text/0024-increase-memo-field-size.md

```diff
@@ -0,0 +1,103 @@
+
+This chart illustrates the breakpoints and what the points of maximum memory utilization are:
+
+| size of `ETxOutRecord` | `2048 / (16 + 1 + sizeof(ETxOutRecord)` |
+| ---------------------- | --------------------------------------- |
+| 187                    | 10.0392156863 |
+| 188                    | 9.99024390244 |
+| 207                    | 9.14285714286 |
+| 210                    | 9.02202643172 |
+| 211                    | 8.98245614035 |
+| 213                    | 8.90434782609 |
+| 233                    | 8.192         |
+| 239                    | 8.0           |
+
+While maintaining 46 byte memos, we would be slightly above the threshold for 9. Therefore we are leaving 26 bytes on the table if we continue to use 46 byte memos. By increasing memo size to 66, we would still be 6 bytes below the threshold for 8, even with additional 6 bytes being used for Tokens. This is consistent with previous policy to leave a small fudge factor in case there is something we overlooked or another need that arises.
+
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This expansion of the memo field without immediate usecase could leave inadequate quantities of space in the ETxOutRecord for other use cases, and it can be difficult to reclaim this space, resulting in bloat. 
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+## Alt: Use a smaller memo size
+
+Alternatively we could try reducing the memo size by 3 bytes. This would allow the 6 bytes required for tokens to use the 3 additional bytes unused and the 3 bytes removed from the memo field to be provided to the tokens. This change would impact recoverable transaction history memos which although no particular memo type uses all 44 bytes of the memo data field, does require all 44 bytes to be used between the 3 memo types. This is therefore, not recommended.
+
+## Alt: Increase memo size even further
+
+Increasing memo size to the next item per bucket level would impact the Omap performance, and would not be recommended for this reason. 
+
+Increasing memo size to the limit of allowed size for the EtxOutRecord for this bucket level would leave no room for error in requirements for the tokens and other features which are in development. Therefore it is desireable to leave a small fudge factor.
+
+## Alt: Move to a variable length memo size
+
+This would go against our privacy goals because variable length memo fields would leak meta data and also cause issues when using ORAM.
+
+## Rationale
+
+This is a minimal change that aligns and standardizes our number of items per bucket to 8, and reserves as much space as possible for future uses that we didn't anticipate, before starting to impact the performance of Fog / the cost of running Fog given that we anticipate additional space per ETxOutRecord being used for Tokens. 
```

Comment:
I might add: 64 bytes is a good number because now the memo is big enough to hold a Schnorr-style digital signature (which in some variants is 64 bytes and in some variants is 48 bytes, for 128 bit security)